[2022-04-21 18:00:01,940] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-04-21T12:00:00+00:00 [queued]>
[2022-04-21 18:00:01,946] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-04-21T12:00:00+00:00 [queued]>
[2022-04-21 18:00:01,946] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-04-21 18:00:01,946] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-04-21 18:00:01,946] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-04-21 18:00:01,954] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): enter_virtual_env> on 2022-04-21T12:00:00+00:00
[2022-04-21 18:00:01,957] {standard_task_runner.py:52} INFO - Started process 260439 to run task
[2022-04-21 18:00:01,962] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'matchup', 'enter_virtual_env', '2022-04-21T12:00:00+00:00', '--job-id', '234', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/matchup.py', '--cfg-path', '/tmp/tmpsfs7kfow', '--error-file', '/tmp/tmp2a6dzq38']
[2022-04-21 18:00:01,963] {standard_task_runner.py:77} INFO - Job 234: Subtask enter_virtual_env
[2022-04-21 18:00:01,992] {logging_mixin.py:104} INFO - Running <TaskInstance: matchup.enter_virtual_env 2022-04-21T12:00:00+00:00 [running]> on host 193e33218975
[2022-04-21 18:00:02,018] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=r.chesne@gmail.com
AIRFLOW_CTX_DAG_OWNER=Robin
AIRFLOW_CTX_DAG_ID=matchup
AIRFLOW_CTX_TASK_ID=enter_virtual_env
AIRFLOW_CTX_EXECUTION_DATE=2022-04-21T12:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-21T12:00:00+00:00
[2022-04-21 18:00:02,018] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-04-21 18:00:02,019] {bash.py:158} INFO - Running command: . /app/env/bin/activate
[2022-04-21 18:00:02,025] {bash.py:169} INFO - Output:
[2022-04-21 18:00:02,031] {bash.py:177} INFO - Command exited with return code 0
[2022-04-21 18:00:02,048] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=matchup, task_id=enter_virtual_env, execution_date=20220421T120000, start_date=20220421T180001, end_date=20220421T180002
[2022-04-21 18:00:02,068] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-04-21 18:00:02,092] {local_task_job.py:146} INFO - Task exited with return code 0
